indexer/feature.cpp

```diff
@@ -475,12 +474,59 @@ uint32_t FeatureType::ParseGeometry(int scale)
     }
     m_parsed.m_points = true;
   }
-  return sz;
 }
 
-uint32_t FeatureType::ParseTriangles(int scale)
+FeatureType::GeomStat FeatureType::GetOuterGeometryStats()
+{
+  ASSERT(!m_parsed.m_points, ());
+  CHECK(m_loadInfo, ());
+  size_t const n = m_loadInfo->GetScalesCount();
+  ASSERT_LESS_OR_EQUAL(n, feature::DataHeader::kMaxScalesCount, ());
+  FeatureType::GeomStat res;
+
+  auto const headerGeomType = static_cast<HeaderGeomType>(Header(m_data) & HEADER_MASK_GEOMTYPE);
+  if (headerGeomType == HeaderGeomType::Line)
+  {
+    size_t const count = m_points.size();
+    if (count < 2)
+    {
+      // Outer geometry present.
+      ASSERT_EQUAL(count, 1, ());
+
+      FeatureType::Points points;
+
+      for (size_t ind = 0; ind < n; ++ind)
```

Comment:
Why ind? We always use i for indexes and it for iterators.

Reply:
`ind` seems to be used throughout this file to refer to the scale index specifically, so I used it too to be consistent;
I'm ok to change it to `i` if you think its better still